test/e2e/cmd/run/run.go

```diff
@@ -208,24 +209,32 @@ func (h *helper) initTestSecrets() error {
 	if h.monitoringSecrets != "" {
 		bytes, err := ioutil.ReadFile(h.monitoringSecrets)
 		if err != nil {
-			return err
+			return fmt.Errorf("reading %v: %w", h.monitoringSecrets, err)
 		}
 
 		monitoringSecrets := struct {
 			MonitoringIP   string `json:"monitoringIp"`
 			MonitoringUser string `json:"monitoringUser"`
 			MonitoringPass string `json:"monitoringPass"`
 			MonitoringCa   string `json:"monitoringCa"`
+			APMServerCert  string `json:"apm_server_cert"`
+			APMSecretToken string `json:"apm_secret_token"`
+			APMServerURL   string `json:"apm_server_url"`
```

Comment:
I know this is inconsistent with what we already have, but I found the chosen naming convention a bit unorthodox. Happy to normalize to the established scheme

Reply:
I'm fine with any of `monitoringIp`, `monitoring-ip`, `monitoring_ip` but we should indeed choose a single one 😄

I'd be +1 with keeping your suggestion in this PR (lower_camelcase everywhere), and use the same naming scheme for the older existing `monitoring*` fields (in a follow-up PR or at least an issue?).

Reply:
I created #3419